CONTRIBUTING.rst

```diff
@@ -3,118 +3,111 @@ Contributing to skrub
 
 First off, thanks for taking the time to contribute!
 
-The following is a set of guidelines for contributing to
-`skrub <https://github.com/skrub-data/skrub>`__.
+Below are some guidelines to help you get started.
 
-|
-.. contents::
-   :local:
 
-|
+Have a question?
+----------------
 
-I just have a question
-----------------------
+If you have any questions, feel free to reach out:
 
-We use GitHub Discussions for general chat and Q&As. `Check it
-out! <https://github.com/skrub-data/skrub/discussions>`__
+- Join our community on `Discord <https://discord.gg/ABaPnm7fDC>`_ for general chat and Q&A.
+- Alternatively, you can `start a discussion on GitHub <https://github.com/skrub-data/skrub/discussions>`_.
 
-What should I know before I get started?
-----------------------------------------
+What to know before you begin
+-----------------------------
 
-To understand in more depth the incentives behind skrub,
-read our `vision statement. <https://skrub-data.org/stable/vision.html>`__
-Also, if scientific literature doesn't scare you, we greatly
-encourage you to read the two following papers:
+To understand the purpose and goals behind skrub, please read our
+`vision statement. <https://skrub-data.org/stable/vision.html>`_
 
-- `Similarity encoding for learning
-  with dirty categorical variables <https://hal.inria.fr/hal-01806175>`__
-- `Encoding high-cardinality string categorical
-  variables <https://hal.inria.fr/hal-02171256v4>`__.
+If you're interested in the research behind skrub,
+we encourage you to explore these papers:
+
+- `Similarity Encoding for Learning with Dirty
+  Categorical Variables <https://hal.inria.fr/hal-01806175>`_
+- `Encoding High-Cardinality String Categorical
+  Variables <https://hal.inria.fr/hal-02171256v4>`_.
 
 How can I contribute?
 ---------------------
 
 Reporting bugs
 ~~~~~~~~~~~~~~
 
-Using the library is the best way to discover new bugs and limitations.
-
-If you find one, please `check whether a similar issue already
-exists. <https://github.com/skrub-data/skrub/issues?q=is%3Aissue>`__
-
-- If so...
+Using the library is the best way to discover bugs and limitations. If you find one,
+please:
 
-  - **Issue is still open**: leave a 👍 on the original message to
-    let us know there are several users affected by this issue.
-  - **Issue has been closed**:
+1. **Check if an issue already exists**
+   by searching the `GitHub issues <https://github.com/skrub-data/skrub/issues?q=is%3Aissue>`_
 
-    - **By a merged pull request** (1) update your skrub version,
-      or (2) the fix has not been released yet.
-    - **Without pull request**, there might be a ``wontfix`` label, and/or a reason at the bottom of the conversation.
+   - If **open**, leave a 👍 on the original message to signal that others are affected.
+   - If closed, check for one of the following:
```

Comment:
Weird, this line appears in bold in the rendered docs, but I don't see why

Reply:
Yes, not sure why either